internal/util/importutilv2/parquet/field_reader.go

```diff
@@ -215,6 +219,35 @@ func ReadStringData(pcr *FieldReader, count int64) (any, error) {
 	return data, nil
 }
 
+func ReadVarcharData(pcr *FieldReader, count int64) (any, error) {
+	chunked, err := pcr.columnReader.NextBatch(count)
+	if err != nil {
+		return nil, err
+	}
+	data := make([]string, 0, count)
+	for _, chunk := range chunked.Chunks() {
+		dataNums := chunk.Data().Len()
+		stringReader, ok := chunk.(*array.String)
+		if !ok {
+			return nil, WrapTypeErr("string", chunk.DataType().Name(), pcr.field)
+		}
+		maxLength, err := parameterutil.GetMaxLength(pcr.field)
```

Comment:
Let’s move the `GetMaxLength` outside the for loop for better performance.

Reply:
My bad, taken it out of the loop.Thanks